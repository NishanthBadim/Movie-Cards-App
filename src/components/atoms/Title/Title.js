import React from 'react';

const Title =(props) => {
    return(
        <h4 className="card-title">
                {props.title}
        </h4>
    );
}
export default Title;