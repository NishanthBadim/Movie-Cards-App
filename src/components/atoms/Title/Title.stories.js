import React from 'react'
import { withKnobs, text } from '@storybook/addon-knobs/react'
import Title from './Title'

export default {
    component: Title,
    title: "Title",
    decorators: [withKnobs]
}

export const Default = () => {
    return (
        <Title title={text("title", "Here goes the Movie Title")} />
    )
}