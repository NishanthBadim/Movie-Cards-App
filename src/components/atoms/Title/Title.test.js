import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Title from './Title'

test("Match the snapshot - Title",() => {
    const input=create(<Title />);
    expect(input.toJSON()).toMatchSnapshot();
})