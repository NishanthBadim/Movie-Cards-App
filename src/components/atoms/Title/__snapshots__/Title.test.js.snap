// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Title 1`] = `
<h4
  className="card-title"
/>
`;
